components/tooltip/tooltip.js

```diff
@@ -812,6 +812,8 @@ class Tooltip extends RtlMixin(LitElement) {
 
 		if (this.showing) {
 			this.updatePosition();
+		} else if (this.getRootNode().activeElement === target) {
```

Comment:
There's also `getComposedActiveElement` is the focus helpers.

Reply:
`getComposedActiveElement` is slightly different since it continually traverses down the DOM until it finds the actual element that has focus. In this case I only want the active element within the context of the tooltip and target's root node.

Reply:
Ah gotcha makes sense to use a quicker short-circuit. 👍